Source/Scene/Cesium3DTileset.js

```diff
@@ -2718,6 +2718,106 @@ Cesium3DTileset.prototype.hasExtension = function (extensionName) {
   return this._extensionsUsed.indexOf(extensionName) > -1;
 };
 
+/**
+ * Check if a property in the <code>3DTILES_metadata.tileset</code> exists.
+ *
+ * @param {String} propertyId The case-sensitive ID of the property.
+ * @returns {Boolean} Whether this property exists.
+ */
+Cesium3DTileset.prototype.hasProperty = function (propertyId) {
```

Comment:
Now that I'm looking at the PR (as opposed to brainstorming offline) I think it might be confusing to have `hasProperty`, `getProperty`, etc be methods of `Cesium3DTileset` because unless someone reads the doc carefully they might not realize what type of metadata this is referring to. For example someone might have features with a `height` property and might expect `tileset.hasProperty("height")` to return `true`, but that would not be the case because this getter is for tileset metadata only.

We should probably go back to the idea of everything being accessed from `tileset.metadata` including the schema and statistics.  `Metadata3DTilesetExtension` should be renamed though if it becomes part of the public API. Maybe `Cesium3DTilesetMetadata`?

Sorry for the churn.

Reply:
no worries, I agree that the `tileset.metadata` usage will be clearer to the user.